docs/getting_started/setup_folder_rights.md

```diff
@@ -0,0 +1,78 @@
+# Setup Folder Rights
+
+Like most things, [Symfony documentation](http://symfony.com/doc/3.4/setup/file_permissions.html) applies here, in this case meaning `var`, `web/var` need to be writable by cli and web server user.
+Furthermore, future files and directories created by these two users will need to inherit those access rights. 
+
+!!! caution
+    For security reasons, in production there is no need for web server to have access to write to other directories.
+
+!!! tip "Development Setup"
+    For development setup you may change your web server config to use same user as owner of folder, what follows below are mainly for production setup, and like Symfony we first and foremost recommend using an approach using ACL.
+
+## A. **Using ACL on a *Linux/BSD* system that supports chmod +a**
```

Comment:
Maybe remove `on a *Linux/BSD* system`? I think the page does not apply to Windows anyway.
Also in other headings below.